docs/HISTORY.txt

```diff
@@ -5,7 +5,10 @@ Changelog
 2.3.1 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Update sliderpane for accessiblity
+  - Show always empty image alt and title attributes on a pane
+  - Remove unused descriptionfield on a pane
+  [elioschmutz]
```

Comment:
I think this is not `rest` valid, can you test that with `longtest` (`zest.releaser`) ?